BUILD.bazel

```diff
@@ -320,6 +320,7 @@ header_template_rule(
         "@CAFFE2_STATIC_LINK_CUDA_INT@": "0",
         "@AT_BUILD_WITH_BLAS@": "1",
         "@AT_BUILD_WITH_LAPACK@": "1",
+        "@AT_BUILD_WITH_SLEEF@": "1",
```

Comment:
These macros generally apply for pytorch codebase. In this case we are trying to use it for specific implementation of an interface. From the code it seems that it is not really used for building sleef but more about using it to 1) include <sleef.h> and 2) use the sleef-code.

So maybe rename this as `AT_BUILD_ARM_VEC256_WITH_SLEEF` (sorry a mouthful) and set it to true by default. That way sleef is used for all oss arm64 builds. For internal fb stuff we will revisit it. @malfet to confirm if thats what he had in mind.

Reply:
Renamed and I have set it to be always enabled.